### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExtensions_ScreenReader:Screenreader does not announce the state as expanded when expand and collapsed when collapses.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52620]
**Environment Details:**
VSCode Version : 1.24.1
OS Version :WIN10
**Additional Details:**
ScreenReaderNVDA[2018.2],JAWS[18.0.5038],MAS1.3.1,Voiceover[MACOS High Sierra 10.13.5 Beta (17F70a)]
**Repro Steps:**
1. Launch VS Code.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Navigate to tree view "Installed" toolbar and navigate to setting button.
4. Hit enter on the button and check the state what screenreader reads.
**Actual:**
Screenreader does not announce the state as expanded when expand and collapsed when collapses.

**Expected:**

Screenreader should announce the state expanded/cpllapsed.
Same issue observed in Jaws and Voiceover also.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not able to know the state whether collapsed or expanded.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference**

[[Accessibility]A11y_VSCode_ViewExtension_NVDA5.pptx](https://github.com/Microsoft/vscode/files/2124155/Accessibility.A11y_VSCode_ViewExtension_NVDA5.pptx)

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52601
